index.js

```diff
@@ -0,0 +1,41 @@
+const booksArr = [];
+const booksIn = document.querySelector('#booksIn');
+
+const add = document.querySelector('#add');
+
+add.addEventListener('click', () => {
+  const title = document.querySelector('#title');
+  const author = document.querySelector('#author');
+  const model = { title: title.value, author: author.value };
+  const getBooks1 = JSON.parse(localStorage.getItem('books'));
+
+  if (!getBooks1) {
+    booksArr.push(model);
+    localStorage.setItem('books', JSON.stringify(booksArr));
+  } else {
+    getBooks1.push(model);
+    localStorage.setItem('books', JSON.stringify(getBooks1));
+  }
+  location.reload(); //eslint-disable-line
+});
+
+const getBooks = JSON.parse(localStorage.getItem('books'));
+if (getBooks === undefined) {
+  booksIn.innerHTML += '<div></div>';
+} else {
+  for (let i = 0; i < getBooks.length; i += 1) {
+    const temp = i;
+    booksIn.innerHTML += `<p>Title : ${getBooks[i].title}</p><p>Author : ${getBooks[i].author}</p><button class="btn-rem" data-id="${temp}">Remove</button>`;
+  }
+}
```

Comment:
- [ ] I highly recommend to fix this console error derives from no books array assigned to the local storage at the page loads first time.   
![image](https://user-images.githubusercontent.com/91486791/204350939-ec6a3e7f-3cda-4a4f-991f-b3153b9317e2.png)